packages/cyberstorm/src/components/Breadcrumb.tsx

```diff
@@ -0,0 +1,86 @@
+import { Fragment, FunctionComponent, ReactNode } from "react";
+
+import { faHouse, faSlashForward } from "@fortawesome/pro-regular-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import styles from "./componentStyles/Breadcrumb.module.css";
+import { IndexLink } from "./Links";
+
+type Label = ReactNode | string;
+
+interface Crumb {
+  label: Label;
+  LinkComponent?: FunctionComponent;
+}
+
+type LinkCrumb<P> = Omit<Crumb, "LinkComponent"> & {
```

Comment:
What is the purpose of `Omit<Crumb, "LinkComponent"> &` here? AFAIK it just picks `label` from `Crumb`, but then redefines it anyway. IMO should be cleared away but not a blocker.

Reply:
Oh, I had left the label there by accident. Idea was to take the base from Crumb and then just redefine `LinkComponent` and add the `LinkProps`.